configs/fastfcn/README.md

```diff
@@ -0,0 +1,12 @@
+# FastFCN: Rethinking Dilated Convolution in the Backbone for Semantic Segmentation
+
+## Reference
+> Huikai Wu, Junge Zhang, Kaiqi Huang. "FastFCN: Rethinking Dilated Convolution in the Backbone for Semantic Segmentation".
+
+## Performance
+
+### ADE20K
+
+| Model | Backbone | Resolution | Training Iters | mIoU | mIoU (flip) | mIoU (ms+flip) | Links |
+|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
+|FastFCN|ResNet50_vd|480x480|120000|-|-|-|-|
```

Comment:
plz add the mIOU results, including flip and ms+flip

Reply:
added.